cookiecutter/wrapper-template/cookiecutter.json

```diff
@@ -0,0 +1,9 @@
+{
+  "source_name": "MyExtensionName",
+  "admin_name": "FirstName LastName",
+  "extension_name": "{{ cookiecutter.source_name.lower() }}",
+  "wrapper_id": "ext-{{ cookiecutter.extension_name }}",
+  "library_name": "{{ cookiecutter.wrapper_id.replace('-', '_') }}",
+  "cli_prefix": "{{ cookiecutter.extension_name.lower() }}",
+  "wrapper_target_name": "some-third-party-cli"
```

Comment:
@aaronsteers right now we've only got one cookiecutter templates to generate a wrapper style util. But we could probably ask a branching question and then use that decide whether or not we need to include a pass_through.py and YOURTHING_invoker command in pyproject.toml